Topic 0/src/abstractFactory/DataBaseType.java

```diff
@@ -0,0 +1,6 @@
+package abstractFactory;
+
+public enum DataBaseType {
+	ORACLE,
+	MYSQL,
```

Comment:
is this compiling? I mean because there is a , at the end. I believe it is an error.

Reply:
This is compiling, and eclipse is throwing no warnings. But i'll remove it.